client/src/cmdhfseos.c

```diff
@@ -1569,8 +1568,8 @@ static int CmdHfSeosManageKeys(const char *Cmd) {
     if (key_index >= 0) {
         operation += 3;
         if (key_index < 4) {
-            if (nonce_len != 0) {
-                PrintAndLogEx(SUCCESS, "Current value for nonce[%d] " _GREEN_("%s"), key_index, sprint_hex_inrow(keys[key_index].nonce, 8));
+            if (keyslot != -1) {
+                PrintAndLogEx(SUCCESS, "Current value for keyslot[%d] " _GREEN_("%u"), key_index, keys[key_index].keyslot);
```

Comment:
Everywhere the keyslot is used internally, it's fetched with a `%02X` format string, but I think it's better displayed to the user in decimal. Is this actually true or should I swap it back to hex everywhere?